views/aboutme.erb

```diff
@@ -0,0 +1,31 @@
+<html>
+  <head>
+    <title>About Me</title>
+    <meta charset = "utf-8">
+  </head>
+
+  <header><h2>Amy Hunter</h2></header>
+
+  <section>
+    <article>I have been learning about programming on my own for the past two years, but never learned enough to make things. I mostly looked at tutorials, attempted a couple of MOOCs, and went to dozens of tech events. Now, I'm in Ada. I write a bunch of code and I am writing programs I couldn't write six weeks ago. </article>
+  </section>
+
+  <section>
+    <p>I am an active member of Seattle Pyladies. Click on the logo to go to our meetup page. Come visit us at a hacknight or event!</p>
+    <p><a href = "http://www.meetup.com/Seattle-PyLadies"><img src = "pyladiespic.jpg"></a></p>
+  </section>
+
+  <section>
+    <p>I also like video games, escpecially platform and puzzle games. I was raised on the NES. Kirby is adorable. Click on Kirby to visit his website!</p>
+    <a href = "http://kirby.nintendo.com/"><img src="Kirby.png" height="100" width="100"></a></p>
+  </section>
+
+  <footer>
+    <uo>
```

Comment:
Yes. I have a tendency to type uo instead of ul for an unordered list for some reason. I will fix it when I start on wave 2.